demos/README.md

```diff
@@ -31,7 +32,8 @@ OpenVINO Model Server demos have been created to showcase the usage of the model
 |[Optical Character Recognition Pipeline](optical_character_recognition/python/README.md) | Run prediction on a JPEG image using a pipeline of text recognition and text detection models with a custom node for intermediate results processing via gRPC API. This demo uses [pipeline](../docs/dag_scheduler.md) with [east_ocr custom node](https://github.com/openvinotoolkit/model_server/tree/develop/src/custom_nodes/east_ocr) and [demultiplexer](../docs/demultiplexing.md) |
 |[Face Detection](face_detection/python/README.md)|Run prediction on a JPEG image using face detection model via gRPC API.|
 |[Single Face Analysis Pipeline](single_face_analysis_pipeline/python/README.md)|Run prediction on a JPEG image using a simple pipeline of age-gender recognition and emotion recogition models via gRPC API to analyze image with a single face. This demo uses [pipeline](../docs/dag_scheduler.md) |
-|[Multi Faces Analysis Pipeline](multi_faces_analysis_pipeline/python)|Run prediction on a JPEG image using a pipeline of age-gender recognition and emotion recogition models via gRPC API to extract multiple faces from the image and analyze all of them. This demo uses [pipeline](../docs/dag_scheduler.md) with [model_zoo_intel_object_detection custom node](https://github.com/openvinotoolkit/model_server/tree/develop/src/custom_nodes/model_zoo_intel_object_detection) and [demultiplexer](../docs/demultiplexing.md) |
+|[Multi Faces Analysis Pipeline](multi_faces_analysis_pipeline/python/README.md)|Run prediction on a JPEG image using a pipeline of age-gender recognition and emotion recogition models via gRPC API to extract multiple faces from the image and analyze all of them. This demo uses [pipeline](../docs/dag_scheduler.md) with [model_zoo_intel_object_detection custom node](https://github.com/openvinotoolkit/model_server/tree/develop/src/custom_nodes/model_zoo_intel_object_detection) and [demultiplexer](../docs/demultiplexing.md) |
+|[Model Ensemble Pipeline](model_ensemble/python/README.md)|Combine multiple models into one pipeline and aggregate results before sending a response. This demo uses [pipeline](../docs/dag_scheduler.md). |
```

Comment:
I would stick to the convention that we focus on the use case in the main demo list. Let's at least mention that we do image classification in that demo:
```
"Combine multiple image classification models into one pipeline and aggregate results to improve classification accuracy"
```

What do you think?

Reply:
done